app/uploaders/ckeditor_picture_uploader.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CkeditorPictureUploader < CarrierWave::Uploader::Base
+  include Ckeditor::Backend::CarrierWave
+
+  # Include RMagick or ImageScience support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+  # include CarrierWave::ImageScience
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
```

Comment:
This method is the same as in https://github.com/IF-115-Ruby/OSBB/blob/5202154b82f2f8bab68c3da40e8de70d65b8c80d/app/uploaders/avatar_uploader.rb#L12, you can inherit from that class